sympy/functions/elementary/tests/test_exponential.py

```diff
@@ -381,7 +381,7 @@ def test_lambertw():
     assert LambertW(2, evaluate=False).is_real
     p = Symbol('p', positive=True)
     assert LambertW(p, evaluate=False).is_real
-    assert LambertW(p - 1, evaluate=False).is_real is None
+    assert LambertW(p - 1, evaluate=False).is_real is False
```

Comment:
@smichr ```eq.as_independent``` working good but sometime ```is_nonpositive``` return ```None``` ans and sometime return ```correct``` ans  for same input.
```
>>> x = Symbol('x', integer=True, negative=True)
>>> (1 - 1/x).is_nonpositive
```
it's return ```None```
```
>>> x = Symbol('x', integer=True, negative=True)
>>> (1 - 1/x).is_nonnegative
True
>>> (1 - 1/x).is_nonpositive
False
```